examples/research_projects/jax-projects/HOW_TO_PROPOSE_PROJECT.md

```diff
@@ -0,0 +1,97 @@
+- 6. *(Optionally) What are possible challenges?*, List possible difficulties with your project. *E.g.* If you know that training convergence usually takes a lot of time, it is worth stating this here!
+- 7. *(Optionally) What is the desired project outcome?* - How would you like to submit the project? How would you like to demo your project? One could *e.g.* create a Streamlit application.
+- 8. *(Optionally) Links to read upon* - Can you provide any links that would help the reader to better understand your project idea?
+
+Feel free to copy-past the following format for your project proposal and fill out the respective sections: 
+
+```
+# <FILL ME: Name of project>
+
+<FILL ME: A clear description of the project>
+
+## 2. Language
+
+The model will be trained in <FILL ME: which language?>.
+
+## 3. Model
+
+<FILL ME: 3. Which model should be used?>
+
+## 4. Datasets
+
+<FILL ME: 4. Which data should be used?>
+
+Possible links to publicly available datasets include:
+- <FILL ME: Link 1 to dataset> 
+- <FILL ME: Link 2 to dataset> 
+- <FILL ME: Link 3 to dataset> 
+
+## 5. Training scripts
+
+<FILL ME: 5. Are there publicly available training scripts that can be used/tweaked for the project?>
+
+We can make use of <FILL ME: link to training script> to train the model.>
+
+## 6. (Optional) Challenges
+
+<(Optionally) FILL ME: 6. What are possible challenges?>
+
+## 7. (Optional) Desired project outcome
+
+<(Optionally) FILL ME: 7. What is the desired project outcome? A demo?>
+
+## 8. (Optional) Reads
+
+The following links can be useful to better understand the project and 
+what has previously been done.
+
+- <FILL ME: Link 1 to read> 
+- <FILL ME: Link 2 to read> 
+- <FILL ME: Link 3 to read> 
+```
+
+To see how a proposed project looks like, please have a look at submitted project 
+proposals [here](https://discuss.huggingface.co/c/flax-jax-projects/22).
+
+## Will my project proposal be selected?
+
+Having submitted a project proposal, you can now promote your idea in the Slack channel `#flax-jax-community-week` to try to convince other participants to join your
+project! Once other people have joined your project, on of the organizers (`@Suzana, @valhalla, @osanseviero, @patrickvonplaten`) will officially create a team for your project and add your project to [this google sheet](https://docs.google.com/spreadsheets/d/1GpHebL7qrwJOc9olTpIPgjf8vOS0jNb6zR_B8x_Jtik/edit?usp=sharing).
```

Comment:
```suggestion
project! Once other people have joined your project, one of the organizers (`@Suzana, @valhalla, @osanseviero, @patrickvonplaten`) will officially create a team for your project and add your project to [this google sheet](https://docs.google.com/spreadsheets/d/1GpHebL7qrwJOc9olTpIPgjf8vOS0jNb6zR_B8x_Jtik/edit?usp=sharing).
```

Reply:
` @Suzana, @valhalla, @osanseviero, @patrickvonplaten`

Are these supposed to be Github, forum, or slack handles?

Reply:
Organizers add project and team members to the spreadsheet or do they have to populate some of it themselves? I think we can take care of it and let them confirm, wdyt?

Reply:
forum handles

Reply:
@suzana-ilic sorry for some reason I didn't see your message before. Yeah I think we can add the groups :-)